packages/frontend/src/components/Routing.js

```diff
@@ -290,44 +291,45 @@ class Routing extends Component {
                                 exact
                                 path='/'
                                 render={(props) => isInactiveAccount ? <ActivateAccountWithRouter {...props}/> : <Wallet tab={tab} setTab={setTab} {...props} />}
-                                accountFound={this.props.account.localStorage?.accountFound}
+                                accountFound={accountFound}
+                                indexBySearchEngines={!accountFound}
                             />
-                            <Route
+                            <PublicRoute
                                 exact
-                                path='/linkdrop/:fundingContract?/:fundingKey?'
+                                path='/linkdrop/:fundingContract/:fundingKey'
                                 component={LinkdropLandingWithRouter}
                             />
-                            <Route
+                            <PublicRoute
                                 exact
                                 path='/create/:fundingContract?/:fundingKey?'
                                 component={CreateAccountWithRouter}
                             />
-                            <Route
+                            <PublicRoute
                                 exact
                                 path={'/create-from/:fundingAccountId'}
                                 component={CreateAccountWithRouter}
                             />
-                            <Route
+                            <PublicRoute
                                 exact
                                 path='/set-recovery/:accountId/:fundingContract?/:fundingKey?'
                                 component={SetupRecoveryMethodWithRouter}
                             />
-                            <Route
+                            <PublicRoute
                                 exact
                                 path='/setup-seed-phrase/:accountId/:step'
                                 component={SetupSeedPhraseWithRouter}
                             />
-                            <Route
+                            <PublicRoute
                                 exact
                                 path='/fund-create-account/:accountId/:implicitAccountId/:recoveryMethod'
                                 component={SetupImplicitWithRouter}
                             />
-                            <Route
+                            <PublicRoute
                                 exact
                                 path='/fund-create-account/success'
                                 component={SetupImplicitSuccess}
                             />
-                            <Route
+                            <PublicRoute
                                 exact
                                 path='/setup-ledger/:accountId'
                                 component={SetupLedgerWithRouter}
```

Comment:
I noticed that the logic for the existing `PrivateRoute` component checks the value of `refreshAccountOwnerEnded`, but that the new version no longer contains that logic.  As a result, we could have changed the behaviour here for these two routes that used the version of the `PrivateRoute` with handling for `refreshAccountOwnerEnded`.

Are you absolutely sure that we can remove the logic for `refreshAccountOwnerEnded` and not break anything about how these two routes function?  If we're not fully confident about this, we should keep the `refreshAccountOwnerEnded` check in place for these two routes to avoid introducing an unintended bug here while cleaning up apparently duplicated logic.

cc: @marcinbodnar Are you familiar with why these two routes care about `refreshAccountOwnerEnded`?